src/isotp_fast.c

```diff
@@ -16,6 +16,20 @@ LOG_MODULE_REGISTER(isotp_fast, CONFIG_ISOTP_LOG_LEVEL);
 static void receive_work_handler(struct k_work *work);
 static void receive_timeout_handler(struct k_timer *timer);
 static void receive_state_machine(struct isotp_fast_recv_ctx *rctx);
+static void send_work_handler(struct k_work *work);
+static void send_timeout_handler(struct k_timer *timer);
+
+/**
+ * Determines whether two @ref isotp_fast_addr structures are equal.
+ */
+static inline bool isotp_fast_addr_equal(struct isotp_fast_addr left, struct isotp_fast_addr right)
```

Comment:
We should use pointers here to avoid copying the structs.

```suggestion
static inline bool isotp_fast_addr_equal(struct isotp_fast_addr *left, struct isotp_fast_addr *right)
```